CockpitPlugin/src/main/java/com/polidea/cockpitplugin/generator/ReleaseCockpitGenerator.kt

```diff
@@ -13,10 +13,12 @@ internal class ReleaseCockpitGenerator : BaseCockpitGenerator() {
     override fun generate(params: List<CockpitParam<*>>, file: File?) {
         val propertyMethods = params.fold(mutableListOf<MethodSpec>()) { acc, param ->
             acc.apply {
-                when (param.value) {
+                val paramName = param.name
+                val value = param.value
```

Comment:
We have "paramName" for param name, but just "value" for param value. Let's make it consistent. :)